charts/alpha/keycloak/values.yaml

```diff
@@ -0,0 +1,45 @@
+keycloak:
+  networkPolicy:
+    enabled: false
+
+  postgresql:
+    enabled: false
+    primary:
+      networkPolicy:
+        enabled: false
+      containerSecurityContext:
+        enabled: false
+
+  externalDatabase:
+    existingSecret: "secret"
+
+  containerSecurityContext:
+    seLinuxOptions: null
+
+  service:
+    headless:
+      annotations:
+        com.palantir.rubix.service/pod-cert: '{}'
+    annotations:
+      com.palantir.rubix.service/pod-cert: '{}'
+
+  tls:
+    enabled: true
+    existingSecret: "cert-{{ .Release.Name }}"
+    truststoreFilename: truststore.jks
+    keystoreFilename: keystore.jks
```

Comment:
we are moving towards opt-in generation of JKS so lets update our use of the cert-api to explicitly request JKS. Let's flup internally on this one.